opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
+
+	// RPCHostname records the host name of the target service
+	RPCHostname = &stringTag{"rpc.hostname"}
+
+	// RPCHostIPv4 records IP v4 host address of RPC target service
+	RPCHostIPv4 = &uint32Tag{"rpc.ipv4"}
+
+	// RPCHostIPv6 records IP v6 host address of RPC target service
+	RPCHostIPv6 = &stringTag{"rpc.ipv6"}
+
+	// RPCPort records port number of RPC target service
+	RPCPort = &uint16Tag{"rpc.port"}
+)
+
+type stringTag struct {
+	Key string
+}
+
+// Add adds a string tag to the `span`
+func (tag *stringTag) Add(span opentracing.Span, value string) {
```

Comment:
not sure how useful it would be, since nobody would usually supply all 5 args, as they overlap. Right now people can set just the tags they have. Also, since this is a standard helper package, as opposed to implementation-specific, it would still need to break apart the 5 args and do 5 individual `set_tag` calls. I think it would make sense to have richer signatures only if we go with an abstract factory pattern.

I see how it might be helpful for source/target separation, but see my comment above about using symmetric tag names for client/server.